pkg/api/bzz.go

```diff
@@ -551,8 +551,7 @@ func (s *Service) serveManifestEntry(
 	mtdt := manifestEntry.Metadata()
 	if fname, ok := mtdt[manifest.EntryMetadataFilenameKey]; ok {
 		fname = filepath.Base(fname) // only keep the file name
-		additionalHeaders[ContentDispositionHeader] =
-			[]string{fmt.Sprintf("inline; filename=\"%s\"", fname)}
+		additionalHeaders[ContentDispositionHeader] = []string{fmt.Sprintf("inline; filename=\"%s\"", fname)}
```

Comment:
I would suggest to use this opportunity to harden the filename handling in this header. It is tricky to deal with a user provided input and even more so in the complex string as this one. If the username has quotes in it it will break the header value parser resulting in a partial filename or no filename at all (depending on browsers and their versions). I would suggest to use the stdlib's approach and add the function escapeQuotes from here https://github.com/golang/go/blob/go1.23.5/src/mime/multipart/writer.go#L154. Going through different RFCs may be overwhelming, and escapeQuotes should be just fine, as long as most browsers, applications and libraries handle the escaping as such.